### PR TITLE
NGI part - TACA

### PR DESCRIPTION
**Pull request by guillermo-carrasco**
I think its easier to review small pull requests, so here a small intro to bioinformatics and to what TACA does.

The `presentation.key` file is just your template, nothing done there yet.

@mariogiov created this flowchart, thanks! I'll just try to make the arrows not cross that much, or split it in several flowcharts for better readability. 

P.S: Yeah... the name of the branch doesn't make much sense..

### CODE REVIEW

File: parts/1-intro/slides.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+
+Bioinformatics is an interdisciplinary field that aims to develop and improve methods
+to retrieve, store, organize and analyze biological data. It encompasses an extensive
+number of research areas, which makes it difficult to precisely define it.
+
+In genomics, the necessity of understanding the DNA structure has driven the development
+of Bioinformatics. Nowadays, 3 days long sequencing experiments on a single machine with the newest sequencing
+technologies can generate as much amount of data as entire genome centers a few years
+ago. This translates into an exponential growth of the information available, which
+created the **need** of well designed and written software to manage and analyze this
```
Comment (robinandeer) [inline parts/1-intro/slides.md]: Maybe mention that today's bottleneck is almost always data processing?

File: parts/2-ngi_pipeline/slides.md [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
-We sequence the source code for life; DNA.
-How do you sequence DNA? What do you do with ACTG?
-Scale of data, automation, backup and clean up, why?
+First and foremost, data needs to go from the sequencer to a machine were the
+preprcessing is done. This is done in local servers because of two reasons.
 
-## TACA
-TACA starts analysis => NGI pipeline
-- configures what analsysis to start
+1. The preprocessing of the data is highly I/O intensive (converting the images
```
Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: I guess you're not going into these details but BCL:s are not technically images, no? More like intensities is what I remembered.

If this is what you meant - please ignore this comment!

File: parts/2-ngi_pipeline/slides.md [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+_NOTE: A slide with a flowchart for TACA will go with this explanation_
+
+**TACA - Tool for the Automated Cleanup and Analysis** is a tool completely written
+in Python 2.7 that does precisely this. When a machine finished sequencing, TACA
+starts automatically the preprocessing of that data in the local servers. To do so
+a cronjob that runs every minute check the status files that the machine generates,
+this is tremendously easy to do in Python.
+
+When the preprocessing of the data is done, the resulting data is sent to our HPC
+(UPPMAX) for further analysis. After sending the data, an HTTP request is sent to the Tornado
```
Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: You might not be showing code examples but at least the infrastructure is very Python-centric! Good :smiley:

**Comment by robinandeer** [comment]
Not much more to say - we can merge it unless you want to make any additional changes?

File: parts/2-ngi_pipeline/slides.md (hunk 2 above)
Comment (guillermo-carrasco) [inline parts/2-ngi_pipeline/slides.md]: yup you're right, but I think its easier to explain: "Convert the pictures that the camera of the sequencer takes into A-C-T-G" than the true story... which I wouldn't be able to explain properly even if I wanted, hehe. 

You think its okay, would you rephrase it?

**Comment by guillermo-carrasco** [comment]
Yay! Can merge if you want, I'll create another small one when I have it :-P

File: parts/1-intro/slides.md (hunk 1 above)
Comment (guillermo-carrasco) [inline parts/1-intro/slides.md]: or storage... :sweat_smile:

File: parts/2-ngi_pipeline/slides.md [hunk 4]
```diff
@@ -1,10 +1,24 @@
-# NGI pipeline
+## TACA - Preprocessing
 
-## Intro to DNA sequencing
-We sequence the source code for life; DNA.
-How do you sequence DNA? What do you do with ACTG?
-Scale of data, automation, backup and clean up, why?
+First and foremost, data needs to go from the sequencer to a machine were the
```
Comment (mariogiov) [inline parts/2-ngi_pipeline/slides.md]: s/were/where/

File: parts/2-ngi_pipeline/slides.md [hunk 5]
```diff
@@ -1,10 +1,24 @@
...
+preprcessing is done. This is done in local servers because of two reasons.
```
Comment (mariogiov) [inline parts/2-ngi_pipeline/slides.md]: s/preprcessing/preprocessing/
Also as long as I'm nitpicking I'd write "on local servers." Maybe I should just PR my edits.

File: parts/2-ngi_pipeline/slides.md [hunk 6]
```diff
@@ -1,10 +1,24 @@
...
 
-## TACA
-TACA starts analysis => NGI pipeline
-- configures what analsysis to start
+1. The preprocessing of the data is highly I/O intensive (converting the images
+that the instrument's camera takes into A-C-T-G string). This makes is non-doable
```
Comment (mariogiov) [inline parts/2-ngi_pipeline/slides.md]: s/is/it/

File: parts/2-ngi_pipeline/slides.md [hunk 7]
```diff
@@ -1,10 +1,24 @@
...
+of mounted or shared filesystems like the ones our HPC is running.
```
Comment (mariogiov) [inline parts/2-ngi_pipeline/slides.md]: s/of/on/

File: parts/1-intro/slides.md (hunk 1 above)
Comment (mariogiov) [inline parts/1-intro/slides.md]: s/of/for/

Comment (robinandeer) [inline parts/1-intro/slides.md]: XD

Can't be easy coming from Spain where  prepositions are such a mess, hehe

(I'll probably regret this comment ...)

**Pull request merged**